packages/strapi-plugin-versioning/README.md

```diff
@@ -0,0 +1,84 @@
+# Strapi - Versioning Plugin (bookshelf only)
+
+Utilities for [Strapi Headless CMS](https://github.com/strapi/strapi) that allow
+searching across nested structures including components.
+
+### ⏳ Installation
+
+(Use **yarn** to install this plugin within your Strapi project (recommended).
+[Install yarn with these docs](https://yarnpkg.com/lang/en/docs/install/).)
+
+```bash
+yarn add strapi-plugin-versioning
+```
+
+Enjoy 🎉
+
+### 🖐 Requirements
+
+Complete installation requirements are exact same as for Strapi itself and can
+be found in the documentation under
+<a href="https://strapi.io/documentation/v3.x/installation/cli.html#step-1-make-sure-requirements-are-met">Installation
+Requirements</a>.
+
+**Supported Strapi versions**:
+
+- Strapi v3.1.1 (recently tested)
+- Strapi v3.x
+
+(This plugin may work with the older Strapi versions, but these are not tested
+nor officially supported at this time.)
+
+**We recommend always using the latest version of Strapi to start your new
+projects**.
+
+## Features
+
+- **Admin entity changes versioning** Now you can are able to check and revert
+  all changes you have made on strapi admin.
+- **Api entity changes versioning:** Also changes made with external api will be
+  visible and reversible
+
+## Setup
+
+For Strapi stable versions, add a middleware.js file within your config folder
+
+```diff
```

Comment:
diff?

Reply:
Github's markdown supports diff when formatting code.

Reply:
Yes I know, because You copied my code, however in that case You are not comparing anything, better will be `bash` or `Shell`